go/webpack/cbor_format_test.go

```diff
@@ -31,10 +31,14 @@ func TestWriteCBOR(t *testing.T) {
 	pack := Package{
 		parts: []*PackPart{
 			&PackPart{
-				url:            staticUrl("https://example.com/index.html?query"),
-				requestHeaders: HTTPHeaders{},
-				status:         200,
+				requestHeaders: HTTPHeaders{
+					httpHeader(":method", "GET"),
+					httpHeader(":scheme", "https"),
+					httpHeader(":authority", "example.com"),
+					httpHeader(":path", "/index.html?query"),
+				},
 				responseHeaders: HTTPHeaders{
+					httpHeader(":status", "200"),
```

Comment:
I think we *do* have uses for other `:status`es: specifically, we should be able to store 301 and 302 redirections in the package. We could make 200 the default if the `:status` is omitted, but in HPACK it's a 1-byte overhead, so I didn't think omitting it was worthwhile.